### ISSUE TITLE
METL

### ISSUE THREAD

**Issue by JamesStrabala** [JamesStrabala/Capstone-Resorces#23]
### Title
Biophysics-based protein language models for protein engineering
### URL
 https://doi.org/10.1101/2024.03.15.585128
### Summary 
Mutational Effect Transfer Learning (METL) is a pretraining strategy to give PLMs physical protein knowledge. It uses molecular models and structure-function information.
Metl has three steps:
1. Synthetic data generation. The developers used Rosetta to model millions of variants, and extracted 55 relevant physical characteristics. 
2. Synthetic data pretraining. An encoder is trained on the information, using a protein-oriented relative position encoder to capture structural information.
3. Experimental data fine tuning. Experimentally-verified structure-function data is input.
Two variants of METL exist, -Local and -Global. -Local is centered around a selected protein, with millions of variants generated with up to 5 substitutions and used for training. -Global takes 148 base proteins and makes 200K variants for each. This method has a significant difference in Spearman correlation- 85% to 16%, meaning it overfits.
METL's proteins were not found to give it an advantage in predicting Rosetta scores. -Local, Linear-Eve, and ProteinNPT outperformed -Global and ESM2 on small datasets. -Global and ESM-2 were on par with small to medium datasets, but ESM-2 pulled ahead as size increased. 
-Local was trained with 64 variants of GFP, making 20 variants for 4 criteria. 
### Citation
@article {Gelman2024.03.15.585128,
	author = {Gelman, Sam and Johnson, Bryce and Freschlin, Chase and D{\textquoteright}Costa, Sameer and Gitter, Anthony and Romero, Philip A.},
	title = {Biophysics-based protein language models for protein engineering},
	elocation-id = {2024.03.15.585128},
	year = {2024},
	doi = {10.1101/2024.03.15.585128},
	publisher = {Cold Spring Harbor Laboratory},
	abstract = {Protein language models trained on evolutionary data have emerged as powerful tools for predictive problems involving protein sequence, structure, and function. However, these models overlook decades of research into biophysical factors governing protein function. We propose Mutational Effect Transfer Learning (METL), a protein language model framework that unites advanced machine learning and biophysical modeling. Using the METL framework, we pretrain transformer-based neural networks on biophysical simulation data to capture fundamental relationships between protein sequence, structure, and energetics. We finetune METL on experimental sequence-function data to harness these biophysical signals and apply them when predicting protein properties like thermostability, catalytic activity, and fluorescence. METL excels in challenging protein engineering tasks like generalizing from small training sets and position extrapolation, although existing methods that train on evolutionary signals remain powerful for many types of experimental assays. We demonstrate METL{\textquoteright}s ability to design functional green fluorescent protein variants when trained on only 64 examples, showcasing the potential of biophysics-based protein language models for protein engineering.Competing Interest Statement: The authors have declared no competing interest.},
	URL = {https://www.biorxiv.org/content/early/2024/03/17/2024.03.15.585128},
	eprint = {https://www.biorxiv.org/content/early/2024/03/17/2024.03.15.585128.full.pdf},
	journal = {bioRxiv}
}
